feed_seeker/feed_seeker.py

```diff
@@ -108,7 +108,7 @@ def default_fetch_function(url):
             return ''
 
     # ConnectionError for 404s, InvalidSchema for email addresses
```

Comment:
looks good -- would you mind updating the comment just above too?

Reply:
Sure thing.